nucypher/characters/lawful.py

```diff
@@ -116,54 +117,97 @@ def generate_kfrags(self, bob: 'Bob', label: bytes, m: int, n: int) -> List:
         delegating_power = self._crypto_power.power_ups(DelegatingPower)
         return delegating_power.generate_kfrags(bob_pubkey_enc, self.stamp, label, m, n)
 
-    def create_policy(self, bob: "Bob", label: bytes, m: int, n: int, federated=False):
+    def create_policy(self,
+                      bob: "Bob",
+                      label: bytes,
+                      m: int,
+                      n: int,
+                      federated: bool = False,
+                      expiration: maya.MayaDT = None,
+                      value: int = None,
+                      handpicked_ursulas: set = None):
         """
         Create a Policy to share uri with bob.
         Generates KFrags and attaches them.
         """
+
+        # Validate early
+        if not federated and not (expiration and value):
+            raise ValueError("expiration and value are required arguments when creating a blockchain policy")
+
+        # Generate KFrags
         public_key, kfrags = self.generate_kfrags(bob, label, m, n)
 
+        # Federated Payload
         payload = dict(label=label,
                        bob=bob,
                        kfrags=kfrags,
                        public_key=public_key,
                        m=m)
 
         if self.federated_only is True or federated is True:
+            # Use known nodes.
+
             from nucypher.policy.models import FederatedPolicy
-            # We can't sample; we can only use known nodes.
             known_nodes = self.known_nodes.shuffled()
             policy = FederatedPolicy(alice=self, ursulas=known_nodes, **payload)
```

Comment:
Now that you've added `handpicked_ursulas`, isn't it odd that it's not used here?

Reply:
Yeah - that's a good suggestion.

In this case - what's are the remaining reasons (*if any*) to include ursulas in `FederatedPolicy.__init__` at all?  It feels a bit like duplicated logic.